services/movie.py

```diff
@@ -0,0 +1,37 @@
+import init_django_orm  # noqa: F401
+
+from django.db.models.query import QuerySet
+from db.models import Movie
+
+
+def get_movies(genres_ids: list[int] = None,
+               actors_ids: list[int] = None) -> QuerySet:
+    filters = {}
+
+    if genres_ids:
+        filters["genres__id__in"] = genres_ids
+
+    if actors_ids:
+        filters["actors__id__in"] = actors_ids
+
+    return Movie.objects.filter(**filters)
+
+
+def get_movie_by_id(movie_id: int) -> QuerySet:
```

Comment:
It returns instance of Movie
```suggestion
def get_movie_by_id(movie_id: int) -> Movie:
```